### PR TITLE
Preview UI supporting multiple choices

### PR DESCRIPTION
**Pull request by nielslyngsoe**
This PR updates the UI of the Preview to accommodate more choices. This is mainly made to enable segments and enable the user to switch segments in Preview.

Here a screenshot of how it's supposed to look:

image

screenshot

Test notes:

- Test dropdowns and switching between different choices.
- Test you can navigate the UI by keyboard.

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/less/canvas-designer.less [hunk 1]
```diff
@@ -14,728 +15,194 @@
...
     overflow: hidden;
+    display: flex;
+    align-items: center;
+    justify-content: center;
     height: 100%;
-    width: 100%;
-    width: calc(~"100% - 80px"); // 80px is the fixed left menu for toggling the different browser sizes
-    position: absolute;
+    height: calc(~"100% - 40px");
+    width: 100%; // 80px is the fixed left menu for toggling the different browser sizes
```
Comment (warrenbuckley) [inline src/Umbraco.Web.UI.Client/src/less/canvas-designer.less]: Can we remove the comment as makes no sense anymore...
```suggestion
    width: 100%;
```

File: src/Umbraco.Web.UI.Client/src/preview/preview.controller.js [hunk 2]
```diff
@@ -136,17 +203,30 @@ var app = angular.module("umbraco.preview", ['umbraco.resources', 'umbraco.servi
...
+                $location.search("culture", iso);
                 setPageUrl();
             }
         };
-
+        /*
         $scope.isCurrentCulture = function(culture) {
             return $location.search().culture === culture;
         }
+        */
```
Comment (warrenbuckley) [inline src/Umbraco.Web.UI.Client/src/preview/preview.controller.js]: Let's not leave in commented in code... we have Git for rollback

File: src/Umbraco.Web.UI/Umbraco/Views/Preview/Index.cshtml [hunk 3]
```diff
@@ -1,72 +1,82 @@
...
+                                    <i class="icon icon-globe-europe---africa"></i><span>@language.CultureName</span>
+                                </button>
+                            }
+                        }
+                    </div>
+                </div>
+
+                <button ng-click="exitPreview()" title="Exit Preview" class="menu-bar__button umb-outline"><svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg"><title>Click to end</title><path d="M5273.1 2400.1v-2c0-2.8-5-4-9.7-4s-9.7 1.3-9.7 4v2a7 7 0 002 4.9l5 4.9c.3.3.4.6.4 1v6.4c0 .4.2.7.6.8l2.9.9c.5.1 1-.2 1-.8v-7.2c0-.4.2-.7.4-1l5.1-5a7 7 0 002-4.9zm-9.7-.1c-4.8 0-7.4-1.3-7.5-1.8.1-.5 2.7-1.8 7.5-1.8s7.3 1.3 7.5 1.8c-.2.5-2.7 1.8-7.5 1.8z"></path><path d="M5268.4 2410.3c-.6 0-1 .4-1 1s.4 1 1 1h4.3c.6 0 1-.4 1-1s-.4-1-1-1h-4.3zM5272.7 2413.7h-4.3c-.6 0-1 .4-1 1s.4 1 1 1h4.3c.6 0 1-.4 1-1s-.4-1-1-1zM5272.7 2417h-4.3c-.6 0-1 .4-1 1s.4 1 1 1h4.3c.6 0 1-.4 1-1 0-.5-.4-1-1-1z"></path><path d="M78.2 13l-8.7 11.7a32.5 32.5 0 11-51.9 25.8c0-10.3 4.7-19.7 12.9-25.8L21.8 13a47 47 0 1056.4 0z"></path><path d="M42.7 2.5h14.6v49.4H42.7z"></path></svg><span>Exit</span>
```
Comment (warrenbuckley) [inline src/Umbraco.Web.UI/Umbraco/Views/Preview/Index.cshtml]: Do we have an SVG sprite we can use rather than inline the SVG?

Comment (nielslyngsoe) [reply src/Umbraco.Web.UI/Umbraco/Views/Preview/Index.cshtml]: oh, actually we do... "icon-power"

**Comment by warrenbuckley** [comment]
Tested and reviewed code, all good

**Pull request merged**